[31m[1mERROR   [0m apilogger:reply_data.py:111 表格写入报错
[31m[1mERROR   [0m apilogger:asserter.py:45 实际结果{"statusMsg": "桌面组名称已存在", "detailErrorReason": "桌面组名称已存在", "statusCode": 1}并不包含期望内容,期望内容是操作成功,失败